source/auth/tests/mongodb-aws.md

```diff
@@ -57,13 +56,13 @@ Configure a custom credential provider to pass valid AWS credentials. The provid
 
 Expect authentication to succeed and the custom credential provider was called.
 
-## Regular credentials
+## Regular credentials *Removed*
```

Comment:
This one isn't removed, it is amended, right?

Reply:
Yes I've fixed this now.